closed/src/java.base/share/classes/sun/security/provider/NativeMD5.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2003, 2014, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
```

Comment:
Perhaps an Oracle copyright notice is inappropriate here?
@pshipton What do you think?

Reply:
It depends. Was any code from OpenJDK copied to this class, or copied and modified? Or is it all freshly written code that has no roots from OpenJDK code?

Reply:
If it's inappropriate here, then it likely shouldn't be in the other subclasses: NativeSHA, NativeSHA2, NativeSHA5. I'm not sure if it belongs in NativeDigest either.

Reply:
I modified `NativeSHA2` (https://github.com/KostasTsiounis/openj9-openjdk-jdk/blob/4df64384b4ba176308e9fe2595a0c812ab490e04/closed/src/java.base/share/classes/sun/security/provider/NativeSHA2.java), so not sure what the appropriate course of action is here.

Reply:
Right, the easiest thing is to leave it as-is. I'm not sure it's a big deal either way.